src/jit/compiler.h

```diff
@@ -2274,7 +2274,11 @@ public :
 #endif // !LEGACY_BACKEND
 
     void                lvaAssignVirtualFrameOffsetsToArgs();
-    int                 lvaAssignVirtualFrameOffsetToArg(unsigned lclNum, unsigned argSize, int argOffs);
+    int                 lvaAssignVirtualFrameOffsetToArg(unsigned lclNum, unsigned argSize, int argOffs
+#ifdef UNIX_AMD64_ABI
```

Comment:
You should look at AMD64_ARG and AMD64_FIRST_ARG in stdmacros.h. Maybe you could add a similar thing for UNIX_AMD64_ABI arguments.